{"primaryContentSections":[{"kind":"declarations","declarations":[{"tokens":[{"kind":"keyword","text":"case"},{"kind":"text","text":" "},{"kind":"identifier","text":"ServerException"}],"languages":["swift"],"platforms":["iOS"]}]}],"schemaVersion":{"major":0,"minor":3,"patch":0},"sections":[],"variants":[{"paths":["\/documentation\/idverification365id\/statustype\/serverexception"],"traits":[{"interfaceLanguage":"swift"}]}],"identifier":{"url":"doc:\/\/com.365id.sdk.idverification\/documentation\/IdVerification365id\/StatusType\/ServerException","interfaceLanguage":"swift"},"abstract":[{"type":"text","text":"Returned if an exception was encountered in the verification process, on the server side."}],"kind":"symbol","metadata":{"fragments":[{"kind":"keyword","text":"case"},{"kind":"text","text":" "},{"kind":"identifier","text":"ServerException"}],"title":"StatusType.ServerException","roleHeading":"Case","role":"symbol","symbolKind":"case","externalID":"c:@M@IdVerification365id@E@StatusType@StatusTypeServerException","modules":[{"name":"IdVerification365id"}]},"hierarchy":{"paths":[["doc:\/\/com.365id.sdk.idverification\/documentation\/IdVerification365id","doc:\/\/com.365id.sdk.idverification\/documentation\/IdVerification365id\/StatusType"]]},"references":{"doc://com.365id.sdk.idverification/documentation/IdVerification365id/StatusType":{"role":"symbol","title":"StatusType","fragments":[{"kind":"keyword","text":"enum"},{"kind":"text","text":" "},{"kind":"identifier","text":"StatusType"}],"abstract":[{"type":"text","text":"Status to be returned after a transaction."}],"identifier":"doc:\/\/com.365id.sdk.idverification\/documentation\/IdVerification365id\/StatusType","kind":"symbol","type":"topic","navigatorTitle":[{"kind":"identifier","text":"StatusType"}],"url":"\/documentation\/idverification365id\/statustype"},"doc://com.365id.sdk.idverification/documentation/IdVerification365id/StatusType/ServerException":{"role":"symbol","title":"StatusType.ServerException","fragments":[{"kind":"keyword","text":"case"},{"kind":"text","text":" "},{"kind":"identifier","text":"ServerException"}],"abstract":[{"type":"text","text":"Returned if an exception was encountered in the verification process, on the server side."}],"identifier":"doc:\/\/com.365id.sdk.idverification\/documentation\/IdVerification365id\/StatusType\/ServerException","kind":"symbol","type":"topic","url":"\/documentation\/idverification365id\/statustype\/serverexception"},"doc://com.365id.sdk.idverification/documentation/IdVerification365id":{"role":"collection","title":"IdVerification365id","abstract":[],"identifier":"doc:\/\/com.365id.sdk.idverification\/documentation\/IdVerification365id","kind":"symbol","type":"topic","url":"\/documentation\/idverification365id"}}}